travis/format.sh

```diff
@@ -50,3 +50,24 @@ if [[ $FAILED_CHECKS -ne 0 ]]; then
   echo "ERROR: Some files are formatted incorrectly. To fix, apply diffs above via patch -p0."
   exit 1
 fi
+
+FILETYPES="*.c *.cc *.cpp *.h *.m *.mm *.dart"
+FILES_TO_CHECK="$(git diff $DIFF_OPTS -- master $FILETYPES)"
+
+for f in $FILES_TO_CHECK; do
+  set +e
+  TRAILING_SPACES="$(grep -P " +$" $f)"
```

Comment:
I'd look for trailing tab characters too.

Reply:
And I'd use "-E" instead of "-P", since that doesn't exist on macOS grep: you can use the blank character class too (which is space and tab).

grep -E '[[:blank:]]+$' .

Reply:
You can avoid both -E and -P with `\s` unless `[[:blank:]]` matches a superset of `\s`.

Reply:
Done. Thanks for the suggestion!